google/cloud/bigtable/table_admin.h

```diff
@@ -166,8 +166,10 @@ class TableAdmin {
             DefaultRPCBackoffPolicy(internal::kBigtableTableAdminLimits)),
         polling_prototype_(
             DefaultPollingPolicy(internal::kBigtableTableAdminLimits)),
-        policies_(bigtable_internal::MakeTableAdminOptions(
-            retry_prototype_, backoff_prototype_, polling_prototype_)) {}
+        options_(google::cloud::internal::MergeOptions(
```

Comment:
Do we need/like qualification beyond `internal::MergeOptions()` (et seq)?

Reply:
`bigtable::internal` is a thing. So I think we need at least `cloud::internal::MergeOptions`.